src/Reader/Http/Response.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Feed\Reader\Http;
+
+use Zend\Feed\Reader\Exception;
+
+class Response implements HeaderAwareResponseInterface
```

Comment:
This is a separated interface, and intended to expose only the features used within `Zend\Feed\Reader\Reader`. It is done to reduce the number of required dependencies, and was done at the request of projects who were wanting to use the component without bringing in additional components.